app/react/Viewer/actions/specs/documentActions.spec.js

```diff
@@ -246,7 +248,8 @@ describe('documentActions', () => {
     describe('getDocument', () => {
       it('should return the document requested', async () => {
         const requestParams = new RequestParams({ sharedId: 'docWithPDFRdy' });
-        const doc = await actions.getDocument(requestParams);
+        const doc = await actions.getDocument(requestParams, 'en', 'filename');
+        expect(doc.documents[0]).toBeDefined();
```

Comment:
@joan, I added this assert to check for the if the doc object is defined or not. Let me know if that's an enough test to check for that

Reply:
@eric i do not see how this is related to the default {} for defaultDoc, actually, if you remove the ```|| {}``` from https://github.com/huridocs/uwazi/blob/2e99e706791f1d370ae2cd0cdb4bccdc81ca2d69/app/react/Viewer/actions/documentActions.js#L98-L100

test is still green, meaning that it is not properly tested.

Reply:
okay, thank you. On it